tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -163,6 +163,19 @@ static cl::opt<bool> SpecConstInfo(
     cl::desc("Display id of constants available for specializaion and their "
              "size in bytes"));
 
+static cl::opt<SPIRV::FPContractMode> FPCMode(
+    "ffp-contract", cl::desc("Fused FP operations:"),
+    cl::init(SPIRV::FPContractMode::On),
+    cl::values(
+        clEnumValN(SPIRV::FPContractMode::On, "on",
+                   "choose a mode according to presence of fused LLVM "
+                   "intrinsics"),
```

Comment:
"fused LLVM intrinsics" look strange. Should it specify what exactly intrinsics affect the mode?

Reply:
Added more information. Let me know if it should be improved further.